lib/chilean/rutify.rb

```diff
@@ -115,8 +122,8 @@ def valid_rut_values?(raw_rut)
       rut = stringify_rut(raw_rut)
       return false unless rut.present?
 
-      rut.chars.each do |i|
-        return false unless valid_rut_value?(i)
+      rut.chars.each_with_index do |i, index|
+        return false unless valid_rut_value?(i, index, rut.length)
       end
```

Comment:
I think it would make more sense to validate more explicitly here than in the unique value method

```suggestion
      rut.chars.each_with_index do |c, i|
        return false unless valid_rut_value?(c, allow_k: i == rut.length - 1)
      end
```